### DISCUSSION TITLE
lawn_mower entity

### DISCUSSION THREAD

**Discussion by Thomas55555** [Thomas55555/husqvarna_automower#538] [Announcements]
There is a new lawn mower entity now in home assistant. I think it's a good idea to migrate this competent from a custom one to an integrated one. Because this fits now really to the entity model and we have tests (thx @prairiesnpr).
So maybe I adapt the custom integration to the lawn mower entity eventually and release it only as beta. So users don't have to migrate twice.

**Comment by Thomas55555** [comment]
Current state:
https://github.com/Thomas55555/core/pull/2

**Comment by robinostlund** [comment]
@Thomas55555 think also a pr to https://github.com/home-assistant/brands needs to be done to add core brands and core integrations 😄

**Reply by Thomas55555** [reply]
It's already done
